src/test/java/com/google/api/codegen/testdata/python_doc_message_library.baseline

```diff
@@ -623,7 +623,7 @@ class Book(object):
       author (string): The name of the book author.
       title (string): The title of the book.
       read (bool): Value indicating whether the book has been read.
-      rating (:class:`google.example.library.v1.library_pb2.Book.Rating`): For testing enums.
+      rating (enum from :class:`google.example.library.v1.enums.Rating`): For testing enums.
```

Comment:
@bjwatson Not sure what the right wording here is -- does this seem sufficient? It's not quite accurate to say that it's an *instance* of `google.example.library.v1.enums.Rating`, since `Rating` is really a container for the enum values.